common/components/componentsBySection/AboutUs/BioModal.jsx

```diff
@@ -14,6 +15,9 @@ type State = {|
   showModal: boolean,
 |};
 
+
+const defaultBiography = "This user hasn't provided us with a biography yet. But trust me, when they do it'll be awesome.";
```

Comment:
Has this text been vetted yet?

Reply:
haha not in the slightest, no. That was my test string for the code, it shouldn't make it to master, good catch.